manager.go

```diff
@@ -249,3 +272,49 @@ func (m *Manager) GetRetries(page uint64, pageSize int64, match string) (Retries
 		RetryJobs:       retryJobs,
 	}, nil
 }
+
+func (m *Manager) startHeartbeat() error {
```

Comment:
It doesn't look like this function really ever returns an error so probably not needed in the function signature.